perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/RegistrarManagerMethod.java

```diff
@@ -601,6 +601,29 @@ public Application call(ApiCaller ac, Deserializer parms) throws PerunException
 
 	},
 
+	/*#
+	 * Creates a new application.
+	 * The method triggers approval for VOs with auto-approved applications.
+	 *
+	 * @param app Application Application JSON object
+	 * @param data List<ApplicationFormItemData> List of ApplicationFormItemData JSON objects
+	 * @return submitted Application
```

Comment:
You must specify data type in return for RPC docs: ``@return Application Submitted application``

Reply:
Fixed